lib/pymedphys/_experimental/chartchecks/dvh_helpers.py

```diff
@@ -0,0 +1,44 @@
+from pymedphys._imports import dicompylercore
+from pymedphys._imports import pandas as pd
+from pymedphys._imports import plotly, plt
+from pymedphys._imports import streamlit as st
+
+
+def plot_dvh(rs_file, rd_file):
+    rs_file = "P:/Share/Chris/BAKER_JEANETTE_61163789/RS.1.3.46.670589.13.15476.20191204085114.491830"
+    rd_file = "P:/Share/Chris/BAKER_JEANETTE_61163789/RD.1.3.46.670589.13.15476.20191204085119.163981"
+    rp_file = "P:/Share/Chris/BAKER_JEANETTE_61163789/RP.1.3.46.670589.13.15476.20191204085117.752796"
```

Comment:
Is this a name that is allowed to be shared?

Reply:
Oops, great catch. This should be removed.

Reply:
Kk, is this the sort of thing that needs to be scrubbed from the GitHub history? (As in, every commit on GitHub has its history stored). It's a bit labour intensive to scrub something from the history... But I can make it happen.

Is that something that is needed here?